docs/browser_support.md

```diff
@@ -1,34 +1,35 @@
 # Browser support
 
 The library supports all popular browsers.
-We use the following terminal command to decide which browsers to support:
+We use the following command to determine which browsers to support:
 
 ```bash
 npx browserslist "cover 96% in us, not IE < 10, not safari < 9"
 ```
 
-At the moment, the browsers are:
+At the moment, these browsers are:
 
-- **Internet Explorer** 11 (see the section below)
+- **Internet Explorer** 11 ([see the section below](#polyfills))
 - **Edge** 18 and 85+
 - **Chrome** 42+
 - **Firefox** 48+
 - **Desktop Safari** 11.1+
 - **Mobile Safari** 9.3+
 - **Samsung Internet** 11.1+
-- **Android Browser** 4.1+ (see the section below)
+- **Android Browser** 4.1+ ([see the section below](#polyfills))
```

Comment:
The `#old-browsers-requirements` target is more appropriate here as well as on line 12

```suggestion
- **Android Browser** 4.1+ ([see the section below](#old-browsers-requirements))
```